daemon.go

```diff
@@ -106,6 +134,14 @@ func newDaemon(ctx context.Context, acceleratedDHT bool) (*daemon, error) {
 		}}, nil
 }
 
+type MutableResolution struct {
```

Comment:
We can kick the can down the road here, but I wonder if adding some explicit versioning to the backend would help us here with upgrades and not feeling bad about shipping something rough and needs to be improved later but that's net an improvement.